app/templates/components/job-tabs.hbs

```diff
@@ -15,14 +15,22 @@
       <LinkTo
         @route="job.config"
         @models={{array this.repo this.job}}
-        @title="Look at this job’s config"
+        @title="Look at this job's config"
         @disabled={{this.job.build.request.noYaml}}
         data-test-build-yaml-tab="true"
       >
         {{#if this.job.build.request.noYaml}}
           <EmberTooltip @text="Try a more recent job to see the config" />
         {{/if}}
+
         View config
+
+        {{#if this.messagesMaxLevel}}
+          <span class="tooltip-wrapper">
+            <SvgImage @name="circle" @class={{this.messagesBadgeIconClass}} />
```

Comment:
I'm not sure about using `svg` here. Just `span` or `div` with `border-radius: 50%` would make the same circle. It's really minor though, so just mentioning it here :)

Reply:
good point, i'll change that :)